src/sentry/static/sentry/app/views/settings/components/dataScrubbing/modals/form/sourceField.tsx

```diff
@@ -288,7 +289,7 @@ class SourceField extends React.Component<Props, State> {
     if (help) {
       if (!isMaybeRegExp) {
         this.setState({
-          help: '',
+          help: t('Where to look. In the simplest case this can be an attribute name.'),
```

Comment:
As this translation is now being used in two different places, it might be a good idea to create a const and reuse it 🙂 Please what do you think?